src/Components/Publishing/Sections/Text.tsx

```diff
@@ -66,7 +90,7 @@ export class Text extends Component<Props, State> {
         postscript={postscript}
       >
         {html.length ? (
-          <div dangerouslySetInnerHTML={{ __html: html }} />
+          <div>{ReactHtmlParser(html, { transform })}</div>
```

Comment:
This converts our html string to JSX where we can apply transforms to each node

Reply:
Following up on standup convo -- we should add a boolean prop for `hasTooltips`, so the parser is only used on appropriate layouts/test outcomes.